build.gradle

```diff
@@ -111,7 +111,9 @@ subprojects {
   targetCompatibility = '1.8'
 
   dependencies {
-    implementation 'org.slf4j:slf4j-api'
+    if (!(project.name.contains("spark") && project.name.contains("runtime"))) {
```

Comment:
maybe `spark-runtime` instead of just `runtime` to make sure only spark-related projects are affected?